proposals/lip-0041.md

```diff
@@ -214,8 +215,8 @@ This command allows users to register a multisignature group for the sender acco
 
```

Comment:
Don't forget to replace networkID by chainID.

Reply:
thanks, fixed